api/v1alpha1/envoygateway_types.go

```diff
@@ -576,6 +576,15 @@ type ExtensionManager struct {
 type ExtensionHooks struct {
 	// XDSTranslator defines all the supported extension hooks for the xds-translator runner
 	XDSTranslator *XDSTranslatorHooks `json:"xdsTranslator,omitempty"`
+
+	// EnablePostTranslateListenersAndRoutes controls whether listeners and routes
```

Comment:
my preference would be to have something like
```
resource:
  listener:
    selector: {}
```

which selects all listeners, these key values are based on resource metadata

or some form with a selector, so we can only send specific resources
* should the top level be `resource` or `rranslationHook` ?

Reply:
wdyt @guydc

Reply:
Created an issue for track: https://github.com/envoyproxy/gateway/issues/6539